### DISCUSSION TITLE
Listening admin api

### DISCUSSION THREAD

**Discussion by Yegor228** [Kong/kong#12602] [Help]
I want Kong to provide access admin api only to the service exposed in the k8s. I'm trying to add a hostname to the admin_listners config.  An error ( Address not available ). Kong versions: 2.7.0, 3.4.0
 Example config: KONG_ADMIN_LISTEN: "hostname:80"

**Comment by hanshuebner** [comment]
Hello @Yegor228,

Are you sure that the hostname that you specify in your `KONG_ADMIN_LISTEN` string resolves to an address that the pod running Kong can bind to?  From what you describe, it does not look like that.
